packages/react/src/components/form/authenticating/index.tsx

```diff
@@ -40,28 +50,44 @@ function Wrapper({ children }: { children: React.ReactNode }) {
   );
 }
 
-export const Authenticating = ({ flowState }: Props) => {
+export const Authenticating = ({ flowState }: Pick<Props, "flowState">) => {
   const factor = flowState.context.config.factor;
+  const isLoggingIn = useRef(false);
+
+  const performLogin = () => {
```

Comment:
`performLogin` is being used as a dependency in `useEffect` so it is safer to create it using `useCallback` to prevent any unwanted effects firing.